bin/install-scalafmt-native.sh

```diff
@@ -1,22 +1,86 @@
 #!/usr/bin/env bash
 set -eux
 
-VERSION="v${1:-2.3.0-RC2}"
+RAW_VERSION=${1:-2.3.0-RC2}
```

Comment:
opt: time to update the default version?

Reply:
Yes, it makes sense